### PR TITLE
Add action link styling

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/19667

## Testing done
Visual. See https://github.com/department-of-veterans-affairs/component-library/pull/51

## Acceptance criteria
- [x] The styling matches [the design](https://app.zeplin.io/project/5e29e7ff922d4d961b5bff4d/screen/603523ff93b3c72f1c9f0298)

### CODE REVIEW

File: packages/formation/sass/full.scss [hunk 1]
```diff
@@ -32,3 +32,4 @@ $image-path: '~uswds/src/img';
 @import 'modules/m-promo-banner';
 @import 'modules/m-responsive-tables';
 @import 'modules/m-back-to-top';
+@import 'modules/m-action-link';
```
Comment (bkjohnson) [inline packages/formation/sass/full.scss:35]: There's another file that we might want to import this into. I'm honestly not sure what the difference is, but the pattern is there, so :shrug:.

**Review by bkjohnson — APPROVED** [review]

File: packages/formation/sass/full.scss (hunk 1 above)
Comment (cvalarida) [reply packages/formation/sass/full.scss:35]: I looked at `core.scss`, but it didn't import the modules. What other file were you thinking of?

Comment (bkjohnson) [reply packages/formation/sass/full.scss:35]: This one: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/site/site.scss

Comment (cvalarida) [reply packages/formation/sass/full.scss:35]: Ah, good call. :+1: Thanks!

**Review by bkjohnson — APPROVED** [review]

File: packages/formation/sass/modules/_m-action-link.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  padding: 8px;
+}
+
+a.va-action-link--blue:before {
+  color: $color-link-default;
+}
+
+a.va-action-link--green:before {
+  color: $color-green;
+}
```
Comment (ncksllvn) [inline packages/formation/sass/modules/_m-action-link.scss:22]: I thought components should use the `vads-c-` prefix? 

> Components have class prefix of c, for component.

https://design.va.gov/components/

**Pull request merged**